centreon/tests/e2e/cypress/e2e/Resources-status/common.ts

```diff
@@ -102,6 +108,16 @@ const insertResourceFixtures = (): Cypress.Chainable => {
   });
 };
 
+const insertDtResources = (): Cypress.Chainable => {
+  const dateBeforeLogin = new Date();
+
+  return cy
+    .setUserTokenApiV1()
+    .then(initializeResourceData)
+    .then(applyConfigurationViaClapi)
+    .then(() => checkThatConfigurationIsExported({ dateBeforeLogin }));
+};
```

Comment:
Since it is only used by downtime tests, put it directly in the test file, or in a dedicated file which is stored in `03-downtime`